app/api/contributors/index.js

```diff
@@ -47,6 +47,13 @@ router.get('/', restricted, async (req, res) => {
 
   const users = (await query).rows;
 
+  let i;
+  for (i = 0; i < users.length; i += 1) {
```

Comment:
Can you move the `let` into the loop body here? Otherwise it doesn't really help with scoping.

`for (let i = 0; i < users.length; i++)`